code/_onclick/xeno.dm

```diff
@@ -59,4 +59,12 @@
 	. = ..()
 	if(!is_charging)
 		return
-	stop_momentum(charge_dir)
\ No newline at end of file
+	stop_momentum(charge_dir)
+
+
+/mob/living/carbon/xenomorph/boiler/ClickOn(atom/A, params)
+	. = ..()
+	if(!selected_ability || !istype(selected_ability, /datum/action/xeno_action/activable/bombard))
```

Comment:
noooooooooooooooooooooooooooo.
use a signal

Reply:
Could you give me an example? This QoL was removed in the ability refactor, and this was my best attempt to add it back.